### DISCUSSION TITLE
Datasource + Dashboard Provisioning - add attribute org_name

### DISCUSSION THREAD

**Discussion by phil-or** [grafana/grafana#39047] [General]
I would like to deploy datasources and dashboards to multiple grafana servers (org_id is different on every grafana node). It would be nice to use the attribute org_name instead of org_id in the provisioning .yaml files.

For notifiers the attribute is available:
notifiers:
```
  - name: notification-channel-1
    type: slack
    uid: notifier1
    # either
    org_id: 2
    # or
    org_name: Main Org.
```

See also:
https://community.grafana.com/t/datasource-provisioning-specify-org-name-rather-than-id/46675

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
